influxdb3_catalog/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license.workspace = true
 # Core Crates
 influxdb-line-protocol.workspace = true
 observability_deps.workspace = true
-schema.workspace = true
+schema = { workspace = true, features = ["v3"] }
```

Comment:
Might be good to keep the feature set at the workspace level, then don't need to set it in each individual cargo file.

Reply:
To clarify, it was previously set in the workspace `Cargo.toml`. Having it there could save someone a few head scratches if they pull one of the `v3` featured crates and find things broken, and I'm not sure we would ever have a crate that depends on one of these and does not use the feature.